vj4/util/validator.py

```diff
@@ -142,3 +143,12 @@ def is_lang(i):
 def check_lang(i):
   if not is_lang(i):
     raise error.ValidationError('lang')
+
+
+def is_domain_invitation_code(s):
+  return bool(DOMAIN_INVITATION_CODE_RE.fullmatch(s))
+
+
+def check_domain_invitation_code(s):
```

Comment:
`check_invitation_code`

and unit test cases, please.

Reply:
The regular expression that function used is so simple that it's correctness doesn't need to be verified.